src/Symfony/Bridge/Twig/Mime/BodyRenderer.php

```diff
@@ -64,34 +62,20 @@ public function render(Message $message): void
 
         if ($template = $message->getTextTemplate()) {
             $message->text($this->twig->render($template, $vars));
+            $message->textTemplate(null);
         }
 
         if ($template = $message->getHtmlTemplate()) {
             $message->html($this->twig->render($template, $vars));
+            $message->htmlTemplate(null);
         }
 
+        $message->context([]);
```

Comment:
This allows a TemplatedEmail to be serialized after rendering the body when the context contains non-serializable variables like Doctrine entities.